pallets/parachain-staking/src/lib.rs

```diff
@@ -1375,6 +1390,73 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		/// Notify a collator is inactive during MaxOfflineRounds
+		#[pallet::call_index(29)]
+		#[pallet::weight(
+			// TODO: add the proper benchmark function once it is available
+			T::DbWeight::get().reads_writes(0 as u64, 0 as u64)
```

Comment:
make sure you change this to use your proper benchmark :)

Reply:
Yes, for that we need to run the benchmarks on the proper server to generate the function. I guess we will need to wait for the weights to be generated pre-release. In the meantime I will add some weight value that results safe for the extrinsic.

Reply:
You can run it locally to generate the file, but you're right that we need to run it on proper benchmarking hardware.